src/pam_pkcs11/pam_pkcs11.c

```diff
@@ -410,58 +390,35 @@ PAM_EXTERN int pam_sm_authenticate(pam_handle_t *pamh, int flags, int argc, cons
       }
 
       if (configuration->slot_description != NULL) {
-	rv = wait_for_token_by_slotlabel(ph, configuration->slot_description,
-          login_token_name, &slot_num);
+          rv = wait_for_token_by_slotlabel(ph, configuration->slot_description,
+                                           login_token_name, &slot_num);
       } else if (configuration->slot_num != -1) {
-        rv = wait_for_token(ph, configuration->slot_num,
-                          login_token_name, &slot_num);
+          rv = wait_for_token(ph, configuration->slot_num,
```

Comment:
this patch is only indentation change.
Really needed?

Reply:
As a part of a larger changes, why not?